src/rust/ide/lib/view/src/graph_editor/builtin/visualization/native/documentation.rs

```diff
@@ -0,0 +1,208 @@
+//! Documentation view visualization generating and presenting Enso Documentation under
+//! the documented node.
+
+use crate::prelude::*;
+
+use crate::graph_editor::component::visualization;
+
+use enso_frp as frp;
+use ensogl::display::DomSymbol;
+use ensogl::display::scene::Scene;
+use ensogl::display;
+use ensogl::system::web;
+use ensogl::system::web::StyleSetter;
+use ast::prelude::FallibleResult;
+
+
+// =================
+// === Constants ===
+// =================
+
+pub const DOC_VIEW_WIDTH  : f32 = 300.0;
+pub const DOC_VIEW_MARGIN : f32 = 15.0;
+
+/// Content in the documentation view when there is no data available.
+const PLACEHOLDER_STR : &str = "<h3>Documentation Viewer</h3><p>No documentation available</p>";
+const CORNER_RADIUS   : f32  = crate::graph_editor::component::node::CORNER_RADIUS;
+
+/// Gets documentation view stylesheet from a CSS file.
+///
+/// TODO TODO TODO TODO TODO TODO TODO TODO TODO TODO TODO TODO TODO TODO TODO
+///      The imported stylesheet is undergoing drastic changes
+///      Expect them to land with https://github.com/enso-org/ide/issues/709
+pub fn doc_style() -> String {
+    format!("<style>{}</style>", include_str!("documentation/style.css"))
+}
+
+
+
+/// Model of Native visualization that generates documentation for given Enso code and embeds
+/// it in a HTML container.
+#[derive(Clone,CloneRef,Debug)]
+#[allow(missing_docs)]
+pub struct ViewModel {
```

Comment:
should this struct be in the constants section? Please review all files and be sure that everything is in right section